vendor/bat-native-ads/src/bat/ads/internal/geographic/subdivision/subdivision_targeting_unittest.cc

```diff
@@ -0,0 +1,110 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/geographic/subdivision/subdivision_targeting.h"
```

Comment:
Could we add tests for when the server responds with invalid JSON and empty country and region